docs/EN/operate/build.md

```diff
@@ -432,3 +461,45 @@ To open Xcode, type `xed .` in the Terminal.
 
 Return to [Open Terminal](#open-terminal).
 
+## Special Section on Trio App Group
+
+* If you have never built Trio before - just follow the instructions on this page: [return to top](#)
+* If you have already built Trio and have added the Trio App Group: [return to top](#)
+* If you have already built Trio, but you did not add the Trio App Group (or are not sure), please read this section for hints and help
+
+Click link for [Mac Build after App Group Change](#mac-build-after-app-group-change).
```

Comment:
Maybe update to:

Instructions for [GitHub](#browser-build-after-app-group-change) | [Mac-Xcode](#mac-build-after-app-group-change)

```
Instructions for [GitHub](#browser-build-after-app-group-change) | [Mac-Xcode](#mac-build-after-app-group-change)
```